wire-library/wire-tests/src/commonTest/kotlin/com/squareup/wire/IgnoreJs.kt

```diff
@@ -1,6 +1,4 @@
 package com.squareup.wire
 
-@OptionalExpectation
```

Comment:
`OptionalExpectation` doesn't work well with dynamically included targets, so better get rid of it and use no-op actuals.